Tiltfile

```diff
@@ -248,21 +242,13 @@ def enable_provider(name, debug):
 
     if profiler_port != 0:
         port_forwards.append(port_forward(profiler_port, 6060))
-        entrypoint.extend(["--profiler-address", ":6060"])
         links.append(link("http://localhost:" + str(profiler_port) + "/debug/pprof", "profiler"))
 
-    # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
-    # build into the container.
```

Comment:
nit: I think we want to keep the comment as it actually belong to the docker-build below